### ISSUE TITLE
'ResponseError' object has no attribute 'message'

### ISSUE THREAD

**Issue by Basalex** [Grokzen/redis-py-cluster#278]
nodemanager.py - > line 180. ResponseError object tries to use message attribute, but message attribute was mark deprecated in 2.6 and was removed in further versions

**Comment by Grokzen** [comment]
@Basalex Can you print a full stack trace of the problem and describe the environment and code that you are trying to run?

**Comment by Basalex** [comment]
Traceback (most recent call last):
   File "/usr/local/lib/python3.6/site-packages/rediscluster/nodemanager.py", line 174, in initialize
     cluster_slots = r.execute_command("cluster", "slots")
   File "/usr/local/lib/python3.6/site-packages/redis/client.py", line 668, in execute_command
     return self.parse_response(connection, command_name, **options)
   File "/usr/local/lib/python3.6/site-packages/redis/client.py", line 680, in parse_response
     response = connection.read_response()
   File "/usr/local/lib/python3.6/site-packages/redis/connection.py", line 629, in read_response
     raise response
 redis.exceptions.ResponseError: This instance has cluster support disabled
 
 During handling of the above exception, another exception occurred:
 
 Traceback (most recent call last):
   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
     response = get_response(request)
   File "/usr/local/lib/python3.6/site-packages/django/utils/deprecation.py", line 93, in __call__
     response = self.process_request(request)
   File "/usr/local/lib/python3.6/site-packages/django/contrib/sessions/middleware.py", line 20, in process_request
     request.session = self.SessionStore(session_key)
   File "/usr/local/lib/python3.6/site-packages/django/contrib/sessions/backends/cache.py", line 17, in __init__
     self._cache = caches[settings.SESSION_CACHE_ALIAS]
   File "/usr/local/lib/python3.6/site-packages/django/core/cache/__init__.py", line 80, in __getitem__
     cache = _create_cache(alias)
   File "/usr/local/lib/python3.6/site-packages/django/core/cache/__init__.py", line 55, in _create_cache
     return backend_cls(location, params)
   File "/usr/local/lib/python3.6/site-packages/redis_cache/backends/single.py", line 21, in __init__
     client = self.create_client(server)
   File "/usr/local/lib/python3.6/site-packages/redis_cache/backends/base.py", line 189, in create_client
     connection_pool = pool.get_connection_pool(client, **kwargs)
   File "/usr/local/lib/python3.6/site-packages/redis_cache/connection.py", line 69, in get_connection_pool
     pool = connection_pool_class(**kwargs)
   File "/usr/local/lib/python3.6/site-packages/rediscluster/connection.py", line 145, in __init__
     self.nodes.initialize()
   File "/usr/local/lib/python3.6/site-packages/rediscluster/nodemanager.py", line 180, in initialize
     if 'CLUSTERDOWN' in e.message or 'MASTERDOWN' in e.message:
 AttributeError: 'ResponseError' object has no attribute 'message'

**Comment by Grokzen** [comment]
@Basalex I guess that e.message was removed on python 3.6 then? So i guess this is a compat issue during those types or errors. I will investigate further.

**Comment by Basalex** [comment]
@Grokzen, looks like it was removed in 3.0 https://www.python.org/dev/peps/pep-0352/#transition-plan

```
Python 3.0 [done]
drop everything that was deprecated above:
string exceptions (both raising and catching) [done]
all exceptions must inherit from BaseException [done]
drop __getitem__, message [done]
```


Guess why tests show no errors.

**Comment by Grokzen** [comment]
Interesting, will have a closer look at this.

**Comment by joshuahaertel** [comment]
`str(e)` instead of `e.message` should do the trick

**Comment by seanpar203** [comment]
I can't believe this hasn't been fixed when it's like 1 line of code. I'll fix this once i'm done with work today.

**Comment by imnotjames** [comment]
@seanpar203 I have a PR open for it if it wasn't obvious from the spam on this issue. ( #307 )

**Comment by Grokzen** [comment]
Resolved in the 3.0 branch. Clsoing for now
